### PR TITLE
Fix Nextcloud 503 Error

### LINKED ISSUE #324: Nextcloud - 503 error
Hi all!
Just to bring an issue.
I had a functional archive on a Nextcloud server using the desktop app. But after a fresh install of Xubuntu, I have systematically 503 errors when trying to save a new pass and randomly 503 errors when trying to open the archive.

### PR DESCRIPTION
**Pull request by sallar**
Fixes #324

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -126,7 +126,7 @@
     "babel-register": "^6.18.0",
     "babili-webpack-plugin": "^0.1.1",
     "buttercup-importer": "~0.9.2",
-    "buttercup-web": "~0.32.1",
+    "buttercup-web": "^0.39.0",
```
Comment (perry-mitchell) [inline package.json]: Probably safer to have this patch-fixed: `~0.39.0`..

File: src/renderer/index.js [hunk 2]
```diff
@@ -20,8 +20,11 @@ import Root from './containers/root';
 const unhandled = require('electron-unhandled');
 unhandled();
 
-// Make crypto faster!
+// Alter some Buttercup internals
 Buttercup.Web.HashingTools.patchCorePBKDF();
+Buttercup.vendor.webdavFS.setFetchMethod(window.fetch);
```
Comment (perry-mitchell) [inline src/renderer/index.js]: Did this work?

Comment (sallar) [reply src/renderer/index.js]: Yes it does :) @perry-mitchell

File: package.json (hunk 1 above)
Comment (sallar) [reply package.json]: @perry-mitchell done mate

File: src/renderer/index.js (hunk 2 above)
Comment (perry-mitchell) [reply src/renderer/index.js]: 😁

**Pull request merged**